api/src/report.controller.ts

```diff
@@ -18,9 +19,10 @@ export class ReportController {
   @Get('/api/report?')
   getReports(
     @PaginateQuery query: PaginationQueryDto,
-    @Query('nick') nick: string,
```

Comment:
Initial idea to create some decorator PaginationFilters or something and use it like that:
```ts
  @Get('/api/report')
  getReports(
    @PaginateQuery query: PaginationQueryDto,
    @PaginationFilters filters: PaginationFilter<Report>, // or `filters: ReportPaginationFilter`
  ): Promise<Paginable<ReportForList>> {
    return this.appService.getList(query,  filters ); // filters.nick and other fields in it
  }
```
But I don't remember when we wanted to do it, here in the current task?